test/test_google_geocoder.rb

```diff
@@ -116,6 +116,23 @@ def test_google_sublocality
      end
    end
 
+   def test_google_administrative_area_level_3
+     @address = "8 Barkwood Lane, Clifton Park, NY 12065"
+     VCR.use_cassette('google_administrative_area_level_3') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.